benchmarks/fastify/startup.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+const benchmark = require('benchmark')
+const suite = new benchmark.Suite()
+
+const Fastify = require('fastify')
+
+suite.add('raw startup', function (deferred) {
+  const server = Fastify()
+  server.ready(() => {
+    deferred.resolve()
+  })
```

Comment:
I do not think this is a valuable metric. Statup time is usually identified as "Time to listening socket" - the time in which it can receive the first request.

This is composed of:

1. time to require the module
2. time to instantiate the framework
3. time that takes `.listen()` to complete

Given that V8 optimized 1 and 2, you'd need to use separate processes (or maybe worker thread) to measure it.

Reply:
Great! Yes, I just create the draft to discuss this kind of measurement. I'm a bit confused on worker_thread for this proposal, the goal with that does just not optimize the startup functions on v8? (to get more reliable metrics).

Perhaps, `eval(...args)` can help us with that. Or, in the last case, we use a different node binary for that. What do you think?

Reply:
I would recommend using child process fork() or worker threads.

Reply:
Yes, exactly!